browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -149,6 +149,93 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_BRAVE_SHIELDS_HTTPS_EVERYWHERE_SWITCH" desc="Switch for HTTPS Everywhere.">
         Upgrade connections to HTTPS
       </message>
+      <message name="IDS_NEXT_TIP_DATE"  desc="Text for Next tip date">
+        Next tip date
+      </message>
+      <message name="IDS_NOTE_TEXT"  desc="Text for note">
+        Note:
+      </message>
+      <message name="IDS_SET_MONTHLY_TIP"  desc="Text for Set monthly tip">
+        Set monthly tip
+      </message>
+      <message name="IDS_ONE_TIME_TIP_AMOUNT"  desc="Text for Next tip date">
+        One-time tip amount
+      </message>
+      <message name="IDS_ONE_TIME_TIP_SUCCESS_NOTE"  desc="Text for one time tip success note">
+        Your tip can take several minutes to process
+      </message>
+      <message name="IDS_CUSTOM_TIP_AMOUNT" desc="Custom tip that allow other amounts">
+        Custom tip amount
+      </message>
+      <message name="IDS_LEARN_MORE_TIP" desc="Learn more tipping">
+        Learn more.
+      </message>
+      <message name="IDS_THANKS_FOR_SUPPORT" desc="Text for Thanks for the support">
+        Thanks for the support!
+      </message>
+      <message name="IDS_YOUR_ONE_TIME_TIP_HAS_BEEN_SENT"  desc="Text for Your one time tip has been sent">
+        Your one-time tip has been sent.
+      </message>
+      <message name="IDS_YOUR_MONTHLY_TIP_HAS_BEEN_SENT"  desc="Text for Monthly tip has been sent">
+        Monthly tip has been set.
+      </message>
+      <message name="IDS_TWEET_ABOUT_YOUR_SUPPORT"  desc="Text for Tweet about your support">
+        Tweet about your support
+      </message>
+      <message name="IDS_MONTHLY_CONTRIBUTION"  desc="Text for Monthly">
+        Monthly
+      </message>
+      <message name="IDS_BAT_FUNDS"  desc="Text for BAT Funds">
+        BAT Funds
+      </message>
+      <message name="IDS_PROCESSING_FEE"  desc="Text for Brave collects 5% of the tip amount as a processing fee">
+        Brave collects 5% of the tip amount as a processing fee.
+      </message>
+      <message name="IDS_CUSTOM_AMOUNT" desc="Text for Custom Amount">
+        Custom Amount
+      </message>
+      <message name="IDS_BAT" desc="Text for BAT">
+        BAT
+      </message>
+      <message name="IDS_USD" desc="Text for USD">
+        USD
```

Comment:
the same as above `translateable="false"`

Reply:
updated